src/app/content/containers/Section.tsx

```diff
@@ -0,0 +1,77 @@
+import { Box, Divider, Heading, Skeleton, Spinner } from '@chakra-ui/react';
+
+import { ClassScheduleListing, Seat, Term, useSeats, useSections } from '../../../fetchers';
+import { SectionInfo } from '../components/Section';
+
+function Sections({ sections, seats }: { sections: ClassScheduleListing[]; seats?: Seat[] | null }): JSX.Element {
+  return (
+    <>
+      {sections.map(({ sectionType, crn, sectionCode, instructionalMethod, additionalNotes, meetingTimes }) => (
+        <SectionInfo
+          section={sectionType}
+          crn={crn}
+          key={crn}
+          sectionCode={sectionCode}
+          instructionalMethod={instructionalMethod}
+          additionalNotes={additionalNotes}
+          meetingTimes={meetingTimes}
+          seat={seats?.find((e) => e.crn === crn)}
+        />
+      ))}
+    </>
+  );
+}
+
+export interface SectionsContainerProps {
+  term: Term;
+  subject: string;
+  code: string;
+}
+
+export function SectionsContainer({ term, subject, code }: SectionsContainerProps): JSX.Element {
+  const { data: sections, loading: dataLoading } = useSections({ term, queryParams: { subject, code } });
+  const { data: seats } = useSeats({ term, queryParams: { subject, code } });
+
+  if (dataLoading) {
+    <Spinner size="sm" />;
```

Comment:
is this ever true? When I run the page I'm not seeing a spinner even though it's loading enough that you'd think it would show up.